### PR TITLE
(#6716) Update video placeholders

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description
- updated video placeholder images for videos
- compressed fbvideochat images since I was in there

## Issue / Bugzilla link
#6716 

## Testing
- Do the image show up?
- Are they on the right videos?

### CODE REVIEW

**Comment by alexgibson** [comment]
The contrast on the video play icon for the top video is a bit harder to see with the new placeholder image. Could people miss that this is a clickable element?

Before:

screenshot_2019-01-16 the new fast browser for mac pc and linux firefox

After:

screenshot_2019-01-16 the new fast browser for mac pc and linux firefox 1

**Comment by alexgibson** [comment]
I can't remember why this video is different to the rest in the way it uses a custom placeholder overlay, and not a native one. There are probably historical reasons why it's like this, but I'm not sure those reasons still exist. Could removing the custom placeholder and making the top video uniform with the other videos be a simple solution? (less code, too).

**Comment by stephaniehobson** [comment]
OMG yes. I assumed it was for a good reason but if there isn't one I'll update it :)

**Comment by stephaniehobson** [comment]
Updated to remove custom placeholder html, script, and test for speed video.

File: bedrock/firefox/templates/firefox/home.html [hunk 1]
```diff
@@ -136,19 +136,14 @@ <h2 class="section-title">{{ _('2x Faster') }}</h2>
...
-                  <source src="https://assets.mozilla.net/video/firefox-home/quantum-vs-chrome.webm" type="video/webm">
-                  <source src="https://assets.mozilla.net/video/firefox-home/quantum-vs-chrome.ogv" type="video/ogg; codecs='theora, vorbis'">
-                  <source src="https://assets.mozilla.net/video/firefox-home/quantum-vs-chrome.mp4" type="video/mp4">
-                  <a href="https://www.youtube.com/watch?v=YIywpvHewc0">
-                    <img src="{{ static('img/firefox/home/quantum-vs-chrome-poster.png') }}" alt="">
-                  </a>
-                </video>
-              </div>{#-- /.video-container --#}
-            </div>{#-- /.moz-video-container --#}
+            <video preload="none" muted controls playsinline poster="{{ static('img/firefox/home/quantum-vs-chrome-poster.png') }}" data-ga-label="Quantum vs Chrome Video"
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/home.html]: This video has audio. Not sure if adding `muted` here is intentional?

Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/home.html]: 🍝

**Comment by stephaniehobson** [comment]
I don't think I deleted `fbvideochat-screenshot.jpg`. I did run it and `fbvideochat-screenshot-high-res.jpg` through tinypng to get a bit of extra compression.

**Comment by stephaniehobson** [comment]
Updated to remove mute and test.

**Pull request merged**